pennylane/qnn/keras.py

```diff
@@ -216,8 +216,12 @@ def __init__(
             self._signature_validation(qnode, weight_shapes)
             self.qnode = to_tf(qnode, dtype=tf.keras.backend.floatx())
 
-        # Allows output_dim to be specified as an int, e.g., 5, or as a length-1 tuple, e.g., (5,)
-        self.output_dim = output_dim[0] if isinstance(output_dim, Iterable) else output_dim
+        # Allows output_dim to be specified as an int or as a tuple, e.g, 5, (5,), (5, 2), [5, 2]
+        # Note: Single digit values will be considered an int and multiple as a tuple
```

Comment:
Is this for strings? E.g. `"52"` is `(5, 2)`? If so, I'm not sure if the note has potential to add confusion, since we don't expect strings at this point.

Reply:
This is just to make sure that anyone who reads this understands that `(5, )` will be passed as an integer and not a tuple. I'll clarify it with an example in there.